src/Files.App/Utils/Storage/Helpers/StorageFileExtensions.cs

```diff
@@ -297,7 +297,7 @@ private static PathBoxItem GetPathItem(string component, string path)
 			{
 				title = "SidebarNetworkDrives".GetLocalizedResource();
 			}
-			else if (component.Contains(':', StringComparison.Ordinal))
+			else if (component.EndsWith(":", StringComparison.Ordinal))
```

Comment:
I think it's better to use the `char` overload.

`component.EndsWith(':')`

Reply:
```suggestion
			else if (component.EndsWith(':'))
```

Reply:
Forgot about the `StringComparison.Ordinal` at my example 💀, you can keep it if you want, but I think it make no difference.

Reply:
Comparison by char is done in `Ordinal`, so I've modified the code to be simpler as you said👍